### PR TITLE
Changes to man page, and include in Makefile

### PR DESCRIPTION
**Pull request by Boruch-Baum**
Includes the man page in the makefile (and changes permissions for icons). Renames the man page to match the program name, and to not overwrite anything else.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,5 +5,7 @@ BINDIR ?= $(PREFIX)/bin
 
 install:
 	@install -Dm755 i3lock-fancy  -t $(DESTDIR)$(BINDIR)
-	@install -Dm755 icons/*      -t $(DESTDIR)$(SHRDIR)/$(PROGNM)/icons
-	@install -Dm644 LICENSE      -t $(DESTDIR)$(SHRDIR)/licenses/$(PRGM)
+        @mkdir   -m755                   $(DESTDIR)$(SHRDIR)/$(PRGM)/icons
```
Comment (AladW) [inline Makefile]: Use tabs in `Makefile`s

File: doc/i3lock-fancy.1 [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 .SH NAME
 .B lock
-- Bash script around i3lock for fancy effect
+- Bash script around i3lock for fancy effects
 
 .SH DESCRIPTION
 
-Takes a screenshot of the desktop, blurs the background and adds a lock icon and
-text. This blurred image is then given to \fBi3lock\fR as the lockscreen
+Takes a screenshot of the desktop or uses a provided PNG file, blurs or pixelates it, and adds a lock icon and
```
Comment (AladW) [inline doc/i3lock-fancy.1]: Unrelated change, part of #124

File: doc/i3lock-fancy.1 [hunk 3]
```diff
@@ -54,7 +54,7 @@ Do not fork i3lock after starting.
 
 .TP
 \fB--\fP
-Must be last option. Set command to use for taking a screenshot. Default is
+Specify a custom command to use for taking a screenshot. This must be the final command-line option. The default is
```
Comment (AladW) [inline doc/i3lock-fancy.1]: "final"?

File: Makefile (hunk 1 above)
Comment (Boruch-Baum) [reply Makefile]: Done

File: doc/i3lock-fancy.1 (hunk 3 above)
Comment (Boruch-Baum) [reply doc/i3lock-fancy.1]: Better than 'last', which carries the connotation 'most recent'. Most native English speakers probably wouldn't care either way.

**Comment by AladW** [comment]
b77540eb2bc458c85410b6fcb26a68ab9e1d6432 and parent e128f9d97aff75189233c454d242d474d9fa855e
